### ISSUE TITLE
Add ability to create and view Error Reports for Bills

### ISSUE THREAD

**Issue by benliscio** [daisybill/daisybill_api#80]
```ruby
bill = DaisybillApi::Models::Bill.find(3333)

bill.error_reports # should return paginated collection of error reports

DaisybillApi::Models::ErrorReport.new(
  bill_id: bill.id,
  comment: "Spoke with...",
  name: "Sally",
  error_type: "incorrect_rejection",
  telephone: "888-555-1234",
  call_reference_number: "1234",
  role: "Adjustor"
)
```

All attributes are required except for call_reference_number and role.

error_types:
- submission_not_on_file
- documents_not_on_file
- incorrect_rejection
- other

**Comment by benliscio** [comment]
@coreyleveen

**Comment by benliscio** [comment]
closed by #84
